src/comments.js

```diff
@@ -381,6 +383,20 @@ function handleConditionalExpressionComments(enclosingNode, followingNode, comme
   return false;
 }
 
+function handleAssignmentPatternComments(enclosingNode, comment) {
```

Comment:
`comment.printed` shouldn't have any affect on the output; it's used to do a sanity check afterwards to make sure we printed all the comments. I don't know why it would affect the output.